app/uk/gov/hmrc/accessibilitystatementfrontend/testonly/views/DisplayYamlPage.scala.html

```diff
@@ -0,0 +1,50 @@
+@*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.newtablinkhelper._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.accessibilitystatementfrontend.models._
+@import uk.gov.hmrc.accessibilitystatementfrontend.views.html._
+@import play.twirl.api.HtmlFormat.raw
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.hmrcstandardpage.TemplateOverrides
+
+@this(layout: Layout, hmrcNewTabLinkHelper: HmrcNewTabLinkHelper, govukButton: GovukButton, copyToClipboard: CopyToClipboard)
+@(yaml: String, rows: Int)(implicit request: RequestHeader, messages: Messages)
+
+@layout(pageTitle = "Accessibility Statement Generator",
+    isWelshTranslationAvailable = false,
+    maybeTemplateOverrides = TemplateOverrides(additionalScriptsBlock = Some(copyToClipboard()))
+) {
+    <h1 class="govuk-heading-xl">Accessibility Statement Generator</h1>
+    <div data-module="copy-to-clipboard">
+        @govukButton(Button(
+            classes = "copy-button",
+            id = Some("copy-button"),
+            content = Text("Copy YAML to clipboard")
+        ))
+
+        <textarea type="text" id="yaml-to-copy" class="yaml-to-copy" cols="100" rows="@{rows.toString}">@yaml</textarea>
+    </div>>
+
+    @hmrcNewTabLinkHelper(NewTabLinkHelper(
+        text = "Take me to the validation page",
+        href = Some("/test-only/validate"),
+        classList = Some("govuk-body")
+    ))
```

Comment:
since we have a form above, we could change this to be "preview how statement will look in new tab" where clicking it submits directly to the validator controller (where in my head, is it worth changing how we describe validator to "preview" - where it might validate it, but maybe the intent of user is to preview how it will look) - or we could display the yaml after generating in the validate yaml form page directly - rather than having two separate pages

don't have to do this now, just what-ifs for the ux